Gems/Atom/RHI/Code/Include/Atom/RHI/CommandList.h

```diff
@@ -11,6 +11,7 @@
 #include <Atom/RHI.Reflect/Scissor.h>
 #include <Atom/RHI/DrawItem.h>
 #include <Atom/RHI/DispatchItem.h>
+#include <Atom/RHI/DispatchRaysItem.h>
```

Comment:
Can you remove the forward declaration of DispatchRaysItem (below), since we're including the header now?

Reply:
Done.

It'd be helpful for small improvements or suggestions to not block the PR, that way the change can be made and submit it directly without having to wait for the block to be removed, it helps for different time zones.

Thanks.